Updates the airgap installation guide (#948)
okteto/docs: src/content/self-hosted/manage/air-gapped.mdx

- **okteto/backend:{variables.chartVersion}**
- **okteto/frontend:{variables.chartVersion}**
- **okteto/buildkit:{variables.chartVersion}** or **okteto/buildkit:{variables.chartVersion}-rootless**
- **okteto/registry:{variables.chartVersion}**
- **okteto/pipeline-runner:{variables.chartVersion}** or **okteto/pipeline-runner:{variables.chartVersion}-rootless**
- **okteto/daemon:{variables.chartVersion}**

> - **okteto/ingress-nginx:{variables.chartVersion}**

Reviewer: this image is `okteto/ingress-nginx-chroot`. The nginx chart adds the `-chroot` if `chroot` is enabled in a [helper](https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/templates/_helpers.tpl#L75), so if we want to have all other chroot functions we need to add the `-chroot` to the image

Author: Added, thanks!

</CodeBlock>

_Need help with your installation? [Contact us](https://www.okteto.com/contact/) for expert guidance._

> ## Step 5: Download the Syncthing Binary

Reviewer 2: can we mention both the synchting and the okteto cli binary

> ## Step 5: Download the Syncthing Binary

Reviewer 2: I suggest that we explain that okteto needs the cli and syncthing, where to download them from and where to install them.  [I wrote this in case it helps.](https://github.com/okteto/airgap-installer/blob/main/README.md#how-to-install-okteto-cli-on-an-air-gapped-environment)

```suggestion
## Step 5: Download the client binaries
```

The Okteto CLI requires the [syncthing](https://syncthing.net/) binary to synchronize your developers' code changes to their remote Development Environments.

By default, the Okteto CLI downloads the syncthing binary from GitHub.

> If your developers don't have direct access to GitHub, you can forward the right synching version to them from:

Reviewer 2: ```suggestion
If your developers don't have direct access to GitHub, you can forward the right syncthing version to them from:
```

<CodeBlock language="bash">
  {`https://github.com/syncthing/syncthing/releases/tag/v${variables.syncthingVersion}`}
</CodeBlock>

> and place it to `$HOME/.okteto` after installing the Okteto CLI.

Reviewer 2: ```suggestion
and place it in the `$HOME/.okteto` folder after installing the Okteto CLI.
```
